package.json

```
@@ -18,19 +18,8 @@
     "check:prettier": "prettier --check .",
     "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "build": "tsc --build",
-    "localenv:network:create": "docker network create --driver=bridge --subnet=10.5.0.0/24 --gateway=10.5.0.1 rafiki",
-    "localenv:network:remove": "docker network remove rafiki",
-    "localenv:cloud-nine-wallet": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml",
-    "localenv:happy-life-bank": "docker compose  -f ./localenv/happy-life-bank/docker-compose.yml",
-    "localenv:tigerbeetle": "docker compose -f ./localenv/tigerbeetle/docker-compose.yml --env-file ./localenv/tigerbeetle/.env.tigerbeetle",
-    "localenv:build": "pnpm localenv:tigerbeetle build && pnpm localenv:cloud-nine-wallet build && pnpm localenv:happy-life-bank build",
-    "localenv:start": "pnpm localenv:network:create && pnpm localenv:tigerbeetle up -d && pnpm localenv:cloud-nine-wallet up -d && pnpm localenv:happy-life-bank up -d",
-    "localenv:stop": "pnpm localenv:happy-life-bank down && pnpm localenv:cloud-nine-wallet down && pnpm localenv:tigerbeetle down && pnpm localenv:network:remove",
-    "localenv:dbvolumes:remove": "docker volume rm cloud-nine-wallet_database-data && docker volume rm tigerbeetle_tigerbeetle-data",
-    "localenv:psql:build": "pnpm localenv:cloud-nine-wallet build && pnpm localenv:happy-life-bank build",
-    "localenv:psql:start": "pnpm localenv:network:create && pnpm localenv:cloud-nine-wallet up -d && pnpm localenv:happy-life-bank up -d",
-    "localenv:psql:stop": "pnpm localenv:cloud-nine-wallet down && pnpm localenv:happy-life-bank down && pnpm localenv:network:remove",
-    "localenv:psql:dbvolumes:remove": "docker volume rm cloud-nine-wallet_database-data",
+    "localenv:compose:psql": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml -f ./localenv/happy-life-bank/docker-compose.yml ${BEFORE_MERGED:-}  -f localenv/merged/docker-compose.yml",
```

```suggestion
    "localenv:compose:psql": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml -f ./localenv/happy-life-bank/docker-compose.yml ${BEFORE_MERGED:-} -f ./localenv/merged/docker-compose.yml",
```
just for consistency